File: SALES_TAX/SALES_TAX/ClientApp/src/app/models/sales-item.ts
import { Category } from "./category";

export class SalesItem {

  public id?: number = 0;
  public name?: string = "";
  public price?: number = 0;
  public quantity?: number = 0;
  public total?: number = 0;
  public salesTax?: number = 0;
  public importTax?: number = 0;
  public imported?: boolean = false;
  public itemCategoryId?: number = 0;
  public itemCategory?: Category = null;
  public Invalid?: boolean = false;

}

File: SALES_TAX/SALES_TAX/Models/SalesItem.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace SALES_TAX.Models
{
    public class SalesItem
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public float Price { get; set; }
        public int Quantity { get; set; }
        public float SalesTax { get; set; }
        public float ImportTax { get; set; }
        public decimal Total { get; set; }
        public bool Imported { get; set; }
        public int ItemCategoryId { get; set; }
        public Category ItemCategory { get; set; }

        public SalesItem()
        {
            Price = 0;
            SalesTax = 0;
            ImportTax = 0;
        }
    }
}
